ci-operator/step-registry/rosa/synthetics/e2e/teardown/rosa-synthetics-e2e-teardown-commands.sh

```
@@ -0,0 +1,33 @@
+
+NS_FILE="${SHARED_DIR}/e2e-namespace"
+if [[ ! -f "${NS_FILE}" ]]; then
+  log "No namespace file found, nothing to clean up"
+  exit 0
+fi
+
+NS=$(cat "${NS_FILE}")
+log "Cleaning up namespace: ${NS}"
+
+# --- OCM login + backplane ---
+source /usr/local/cs-qe-credentials/ocm-tokens
```

Shellcheck (0.11.0)

[info] Not following: /usr/local/cs-qe-credentials/ocm-tokens was not specified as input (see shellcheck -x).

(SC1091)